### ISSUE TITLE
[arm][ethos-u55] Add amax support in ET backend

### ISSUE THREAD

**Issue by JakeStevens** [pytorch/executorch#17211]
### 🚀 The feature, motivation and pitch

Per comment on https://github.com/pytorch/executorch/pull/17109, Vela support landed and amax can now be supported in Executorch for u55 too. 

This unblocks a stable softmax implementation

**Comment by AdrianLundell** [comment]
PR here: https://github.com/pytorch/executorch/pull/17372

**Comment by AdrianLundell** [comment]
Merged into main now, please note that you still need to overwrite the default ArmPassPipelineConfig in the compile spec to get the stable softmax algorithm during a transition period before the unstable algorithm is deprecated.:
`compile_spec.set_pass_pipeline_config(ArmPassPipelineConfig())`

**Comment by JakeStevens** [comment]
Great! With this, should we review+merge

https://github.com/pytorch/executorch/pull/17109

Until the unstable is deprecated fully?
